libs/application/ui-fields/src/lib/KeyValueFormField.tsx

```diff
@@ -13,13 +13,16 @@ const KeyValueFormField: FC<{
   application: Application
 }> = ({ field, application }) => {
   const { formatMessage } = useLocale()
+  const values = formatText(field.value as string, application, formatMessage)
 
   return (
     <Box>
       <Text variant="h4">{field.label}</Text>
-      <Text>
-        {formatText(field.value as string, application, formatMessage)}
-      </Text>
+      {Array.isArray(values) ? (
+        values.map((value) => <Text>{value}</Text>)
+      ) : (
+        <Text>{values}</Text>
+      )}
```

Comment:
Still getting back into this project.

Is KeyValueFormField a kind of read-only field? Is it (visually?) better to wrap each value in a Text than use newlines and some handy css?

Reply:
Yeah, KeyValueFormField is a read-only field.
I originally went with the newline, but raised concerns since these values can come from CMS; and therefore changed it to rendering multiple `<Text>`